### PR TITLE
Add vote claim error messages

### LINKED ISSUE #422: Show error details while voting on a claim
**Is your feature request related to a problem? Please describe.**
Users have no feedback when their vote fails

it can have multiple reasons, we log them on sentry but it would be useful to have them displayed somewhere for the user too

**Describe the solution you'd like**
add a collapsible text on the feedback informing the state 

**Describe alternatives you've considered**
ask the design team for alternatives 

**Additional context**
claims are still buggy, we need to interact over it to fix the voting flow

### PR DESCRIPTION
**Pull request by amiskov**
## What issue does this PR close
Closes #422 

## Changes Proposed (a list of new changes introduced by this PR)
After the `verifyclaim` transaction Elm may receive a JS `Value` with the field `error` containing the JSON-string representation of the errors from EOS:
image

Proposed changes are:
- Add a module for parsing JSON-strings with errors from EOS.
- Extract the appropriate message from the EOS errors list and show it to the user as a feedback message when an error occurs.

## How to test (a list of instructions on how to test this PR)
Try to vote some non-valid Claim from the Dashboard, Analysis, or Claim View page and see the error like this:
image

### CODE REVIEW

**Comment by amiskov** [comment]
@lucca65 Can you please have a look at the approach I used in this PR for showing EOS voting errors to the user? This is a work in progress but I would really appreciate your opinion on what's already done.

File: public/translations/pt-BR.json [hunk 1]
```diff
@@ -88,7 +88,14 @@
     "unknown": "Opa. Algo de errado aconteceu =(.",
     "accountNotFound": "Conta não encontrada. Tem certeza que criou uma conta com essa chave?",
     "phone": "Por favor insira um número de telefone válido",
-    "zipcode": "CEP inválido"
+    "zipcode": "CEP inválido",
+    "contracts": {
+      "verifyclaim": {
+        "Can't vote on already verified claim": "Não é possível votar em uma reivindicação já verificada",
+        "This is action is already completed, can't verify claim": "Esta ação já foi concluída, não é possível verificar a reivindicação",
+        "There are no usages left for this action": "Não há usos restantes para esta ação"
```
Comment (lucca65) [inline public/translations/pt-BR.json:96]: we can also add a `error.unknown` for unhandled error cases, with the same general message we have on `master`

**Review by lucca65 — CHANGES_REQUESTED** [review]
@amiskov damn! this is very clever! I loved the approach, just added a suggestion and we can add translations in other languages too

File: public/translations/pt-BR.json (hunk 1 above)
Comment (amiskov) [reply public/translations/pt-BR.json:96]: Yep, I already use `error.unknown` as a [fallback](https://github.com/cambiatus/frontend/blob/32f81c6d47a2636b2d64d637b389761f4ab57937/src/elm/Eos/EosError.elm#L31) while parsing JSON with errors.

Comment (lucca65) [reply public/translations/pt-BR.json:96]: Yeah, just talking about that we have no translations yet (I know its a draft)! nice one!

File: src/elm/Page/Dashboard.elm [hunk 2]
```diff
@@ -408,8 +409,9 @@ viewAnalysis loggedIn claimStatus =
         ClaimVoted _ ->
             text ""
 
-        ClaimVoteFailed _ ->
-            div [ class "text-red" ] [ text "failed" ]
+        ClaimVoteFailed claim ->
+            Claim.viewClaimCard loggedIn claim
+                |> Html.map ClaimMsg
```
Comment (lucca65) [inline src/elm/Page/Dashboard.elm:414]: I love this approach!

File: src/elm/Eos/EosError.elm [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+{-| Extracts failure description from the first error message received from the EOS:
+
+    extractFailure "assertion failure with message: Can't vote on already verified claim"
+        == "Can't vote on already verified claim"
+
+The result is a string which can be translated and showed to the user.
+
+-}
```
Comment (lucca65) [inline src/elm/Eos/EosError.elm:14]: 👏 😭

File: src/elm/Eos/EosError.elm [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                _ ->
+                    -- Keep whole message (may be verbose, with no translation, but better than nothing)
+                    firstMessage
+
+        _ ->
+            "error.unknown"
+
+
+prepareErrorMessage : Translators -> Maybe String -> String
+prepareErrorMessage { t } eosErrorString =
```
Comment (lucca65) [inline src/elm/Eos/EosError.elm:36]: I'm not 100% sure about this name, maybe `translateErrorMessage` or `parseErrorMessage`. The word `prepare` seems out of place, idk

**Review by lucca65 — APPROVED** [review]
Looks good! I think this is a step on the right direction, hopefully this is the new default on handling blockchain errors. We should have this type of approach on all calls

- Community creation
- transfers
- shop buy
- sale creation
- action creation
- claims

This is so good that I added an description on #419

File: src/elm/Eos/EosError.elm (hunk 4 above)
Comment (amiskov) [reply src/elm/Eos/EosError.elm]: Agree! Let's go with `parseErrorMessage`.

**Review by lucca65 — APPROVED** [review]

**Pull request merged**